sessionize/app/src/main/java/co/touchlab/sessionize/settings/SettingsViewModel.kt

```diff
@@ -2,10 +2,12 @@ package co.touchlab.sessionize.settings
 
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.ViewModelProvider
+import co.touchlab.sessionize.ServiceRegistry
 import co.touchlab.sessionize.SettingsModel
+import co.touchlab.sessionize.platform.NotificationsModel
 
 class SettingsViewModel: ViewModel(){
-    val settingsModel = SettingsModel()
```

Comment:
Updated settings model to take in it's dependencies, makes testing and mocking much easier and cleaner